code/core/src/main/java/com/adobe/marketing/mobile/launch/rulesengine/HistoricalEventsQuerying.kt

```diff
@@ -32,7 +32,7 @@ internal fun historicalEventsQuerying(
         var eventCounts = 0
         extensionApi.getHistoricalEvents(
             requests.toTypedArray(),
-            searchType == SEARCH_TYPE_ANY
+            searchType != SEARCH_TYPE_ANY
```

Comment:
nit: To future proof, can you define a new constant and change it to searchType == SEARCH_TYPE_ORDERED.

Reply:
Can you also add a test to test this case?

Reply:
will get these updated